### PR TITLE
Quiz js, index and css

### PR DESCRIPTION
**Pull request by nathantbaker**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <title>The Static Web JavaScript Quiz</title>
+    <link rel="stylesheet" type="text/css" href="styles.css">
```
Comment (gregkorte) [inline index.html]: Check your whitespace.

File: quiz.js [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+// Function to push user inputs into object
+function pushInputs() {
+  // Save new inputs
+  height = heightElement.value;
+  character = characterElement.value;
+  // Error if either of new inputs are nothing
+  if (character == "" || height == "") {
+    alert(alertMissingInput);
+  } else {
+    makeTree(height, character);
```
Comment (gregkorte) [inline quiz.js]: Instead of passing individual variables into this function, you should be passing the entire object.

**Comment by gregkorte** [comment]
Nice quiz, Nate! Good comments, good indentation, Easy to follow code. Just fix your object and whitespace and let me know when you are finished via slack. I'll take another look at it. For bonus consider these questions. What happens if you enter more than one character in your character input? What happens if you enter a letter in the height field?

**Comment by nathantbaker** [comment]
Thanks for the feedback! I updated the repo based on your suggestions

**Comment by gregkorte** [comment]
Looks good, Nate. You're good to merge! :+1:

**Pull request merged**